src/steps/GithubActionStep.ts

```diff
@@ -0,0 +1,25 @@
+import { Step } from 'aws-cdk-lib/pipelines';
+import { JobStep } from '../workflows-model';
+
+interface GithubActionStepProps {
```

Comment:
```suggestion
interface GitHubActionStepProps {
```

Reply:
I wonder there is no difference in your suggested change? ❓

Reply:
capital H :)